src/Slider.jsx

```diff
@@ -122,13 +125,19 @@ export default React.createClass({
       nextProps.orientation !== this.props.orientation
     );
 
+    const disabledChanged = nextProps.disabled !== this.props.disabled;
+
     if (orientationChanged) {
       this.setState({
         className: getClassName(nextProps),
       });
     }
 
     if (minMaxChanged || valuesChanged) this.updateNewValues(nextProps);
+
+    if (disabledChanged) {
+      this.endSlide();
```

Comment:
maybe i'm just missing the context on `endSlide` here - but do we want it to happen whenever "disabled" changes? or just when true goes to false